src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -1109,52 +1286,38 @@ public void testListStreams()  {
                                 "          \"layoutClassList\" : [] \n" +
                                 "          } ]\n" +
                                 "        }")));
-        try {
-            StreamList streams = sdk.listStreams(sessionID);
-            assertNotNull(streams);
-            assertEquals(2,streams.getTotalCount());
-            Stream stream1 = streams.get(0);
-            Stream stream2 = streams.get(1);
-            assertEquals("1234", stream1.getId());
-            assertEquals("", stream1.getName());
-            assertEquals("camera", stream1.getVideoType());
-            assertEquals("5678", stream2.getId());
-            assertEquals("", stream2.getName());
-            assertEquals("screen", stream2.getVideoType());
-
-            verify(getRequestedFor(urlMatching(url)));
-            assertTrue(Helpers.verifyTokenAuth(apiKey, apiSecret,
-                    findAll(getRequestedFor(urlMatching(url)))));
-            Helpers.verifyUserAgent();
-        } catch (Exception e) {
-            exceptionThrown = true;
-        }
-        assertFalse(exceptionThrown);
+        StreamList streams = sdk.listStreams(sessionID);
+        assertNotNull(streams);
+        assertEquals(2,streams.getTotalCount());
+        Stream stream1 = streams.get(0);
+        Stream stream2 = streams.get(1);
+        assertEquals("1234", stream1.getId());
+        assertEquals("", stream1.getName());
+        assertEquals("camera", stream1.getVideoType());
+        assertEquals("5678", stream2.getId());
+        assertEquals("", stream2.getName());
+        assertEquals("screen", stream2.getVideoType());
+
+        verify(getRequestedFor(urlMatching(url)));
+        assertTrue(Helpers.verifyTokenAuth(apiKey, apiSecret,
+                findAll(getRequestedFor(urlMatching(url)))));
+        Helpers.verifyUserAgent();
     }
 
     @Test
     public void testforceDisconnect() throws OpenTokException {
         String sessionId = "SESSIONID";
         String connectionId = "CONNECTIONID";
-        Boolean exceptionThrown = false;
         String path = "/v2/project/" + apiKey + "/session/" + sessionId + "/connection/" + connectionId ;
-
         stubFor(delete(urlEqualTo(path))
                 .willReturn(aResponse()
                         .withStatus(204)
                         .withHeader("Content-Type", "application/json")));
-
-        try {
-            sdk.forceDisconnect(sessionId,connectionId);
-            verify(deleteRequestedFor(urlMatching(path)));
-            assertTrue(Helpers.verifyTokenAuth(apiKey, apiSecret,
-                    findAll(deleteRequestedFor(urlMatching(path)))));
-            Helpers.verifyUserAgent();
-
-        } catch (Exception e) {
-            exceptionThrown = true;
-        }
-        assertFalse(exceptionThrown);
+        sdk.forceDisconnect(sessionId,connectionId);
```

Comment:
If this throws an exception, shouldn't we be testing this with a `try/catch` ?

Reply:
Before I was doing a try catch for all the exceptions our API, JUNIt etc would throw,
Thinking about it later , I felt I should not catch those exceptions but rather let it propagate and have JUNIOt take care of it. (thinking about the case when I catch JUNIT exception but forget to handle them and tests would pass)